src/components/BoxForm/BoxForm.css

```diff
@@ -0,0 +1,145 @@
+:root {
+  --text-gray-color: #808080;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.boxForm {
+  display: grid;
+  grid-gap: 10px;
+  grid-template-columns: repeat(3, 1fr);
+  grid-template-rows: 200px 200px 200px 80px;
+  color: #444;
+  margin: 20px 50px;
+  column-gap: 4rem;
+}
+
+/* CSS Grid setup */
+.a {
```

Comment:
Could we rename these classnames to something more valuable? Maybe name it the section that it is styling.